keyboards/nopunin10did/railroad/readme.md

```diff
@@ -0,0 +1,15 @@
+# The Railroad
+
+![The Railroad](https://i.imgur.com/B8OjXmyl.jpg)
+
+A long keyboard with ortholinear alphas and semi-standard modifier keys. For more information, join the [NoPunIn10Key Discord](https://discord.gg/sku2Y6w).
+
+* Keyboard Maintainer: [Alex Ronke](diplomacyvariants.wordpress.com), also known as NoPunIn10Did. 
+* Hardware Supported: The Railroad: 2020-07 Prototype
+* Hardware Availability: Currently by request only
+
+Make example for this keyboard (after setting up your build environment):
+
+    qmk compile -kb nopunin10did/railroad -km default
```

Comment:
```suggestion
    make nopunin10did/railroad:default
```

Reply:
The official documentation still lists this notation as correct. Is everything moving to make now?

Reply:
No, the other way around. But this is what's currently in the template.

Reply:
So why change it?

Reply:
You are right `qmk compile` is what we are moving forward to, but:

- currently every board has the `make` command

- it's our current template

- it's called "Make example"

Changing all this will probably happen, but it will be a change to all these things and not just one keyboard.